README.md

```diff
@@ -27,6 +27,16 @@
 4. Run `npm i` to install the latest dependencies to your computer.
 5. In a terminal, run `npm start` or, if you want to run the engineering dashboard without the solar car dashboard, `npm run start-dev`. The console output may look messy, but that is due to three servers running at the same time. For a more organized console, please look below.
 
+## Running the Dashboard With Docker
+
+0. Install [Docker Desktop](https://docs.docker.com/get-docker/) or [Docker](https://docs.docker.com/engine/install/). 
+1. If you are not familiar with Docker, it is recommended to install Docker Desktop.
+2. Once it is installed, start Docker Desktop. If docker is not started, you will see a message saying Docker Daemon is Starting. Wait for this to complete.
+3. Look for a script in this GitHub repo that matches your OS. If you're on Windows, that script is called `windows-pull-run.bat`, and for Linux and Mac it is called `linux-pull-run.sh`.
+4. To download the script, open the script in Github and then click "Raw". The file will open in a tab. Then do `Ctrl + S` or `Command + S`, depending on your OS, save it to your Downloads folder, and the script will be downloaded. To run, look at the bullet points below:
+   - (Windows) Run these commands in your command prompt `cd Downloads`, `move windows-pull-run.bat.txt windows-pull-run.bat`, and `windows-pull-run.bat`. 
```

Comment:
To keep out of the terminal, I would just say to remove ".txt" when saving the file. Then, you can just double-click it in the file explorer.

Reply:
updated